Buildasaur/MenuItemManager.swift

```diff
@@ -19,13 +19,12 @@ class MenuItemManager : NSObject, NSMenuDelegate {
         
         let statusItem = statusBar.statusItemWithLength(32)
         statusItem.title = ""
-        let image = NSImage(named: "icon")
-        image?.setTemplate(true)
```

Comment:
This was added because of issues with compatibility with both light and dark menu bar modes. Did you test that it looks good on both even when you disable templating?